src/GafferImage/DeepHoldout.cpp

```diff
@@ -142,10 +142,15 @@ void DeepHoldout::affects( const Gaffer::Plug *input, AffectedPlugsContainer &ou
 		outputs.push_back( intermediateInPlug()->channelNamesPlug() );
 	}
 
-	if( input == flattenedPlug()->channelDataPlug() || input == inPlug()->channelNamesPlug() )
+	if( input == flattenedPlug()->channelDataPlug() )
```

Comment:
We need to keep this line as it was, because `computeChannelData( parent = outPlug() )` does pull on `inPlug()->channelNamesPlug()`.